### PR TITLE
Atom list and single page

### PR DESCRIPTION
**Pull request by ShaunYearStrong**
Atom List page (/video/atoms):

picture 5

Single Atom page (/video/atoms/:id):

picture 6

### CODE REVIEW

File: public/video-ui/src/services/pandaReqwest.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+export function pandaReqwest(reqwestBody) {
+  return Q.Promise(function(resolve, reject) {
+    Reqwest(reqwestBody)
+        .then(res => {
+          resolve(res)
+        })
+        .fail(err => {
+          if (err.status == 419) {
+            const reauthUrl = '/reauth';
```
Comment (clloyd) [inline public/video-ui/src/services/pandaReqwest.js]: I think we need to implement this route?

Comment (ShaunYearStrong) [inline public/video-ui/src/services/pandaReqwest.js]: Oops, right you are!

Comment (ShaunYearStrong) [inline public/video-ui/src/services/pandaReqwest.js]: Fixed in commit 1fbb044

File: app/controllers/VideoUIApp.scala [hunk 2]
```diff
@@ -2,11 +2,15 @@ package controllers
...
 import play.api.mvc._
 import javax.inject._
+
 import com.gu.pandomainauth.action.AuthActions
 
+
 class VideoUIApp @Inject() (val authActions: AuthActions)
   extends AtomController {
 
+  import authActions.{ AuthAction }
```
Comment (clloyd) [inline app/controllers/VideoUIApp.scala]: Did IntelliJ insert that import here? Seems a really odd place

**Pull request merged**